sim/core/buffs.go

```diff
@@ -596,6 +596,14 @@ func registerBloodlustCD(agent Agent) {
 
 func BloodlustAura(character *Character, actionTag int32) *Aura {
 	actionID := BloodlustActionID.WithTag(actionTag)
+
+	sated := character.GetOrRegisterAura(Aura{
```

Comment:
sated was added because we want to be able to control a shaman casting bloodlust in the APL.  If you have the raid buff also selected and have both cast in the APL, we should prevent the second one from working.